api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -174,6 +174,19 @@ message TransformationTemplate {
   // resulting header, the rendered template will determine the value.
   map<string, InjaTemplate> headers = 3;
 
+  // Defines a header-template pair to be used in `headers_to_append`
+  message HeaderToAppend {
+    // Header name
+    string key = 1;
+    // Apply a template to the header value
+    InjaTemplate value = 2;
+  }
+
+  // Use this attribute to transform request/response headers. It consists of 
+  // an array of string/template objects. Use this attribute to define multiple 
+  // templates for a single header.
```

Comment:
Can we explain here that the resulting headers will be added, not replace existing.